Gemfile

```diff
@@ -88,4 +88,6 @@ group :development do
   gem 'mailcatcher', '~> 0.5.12'
   gem 'quiet_assets', '~> 1.1.0'
   gem 'rdoc', '~> 3.12.2'
+  gem 'better_errors'
```

Comment:
>Should add a pessimistic version constraint to all gems

Can this be added to the contribution guide (#2660)? I've never seen this done to every gem in a project before and I'd be genuinely interested in the reasoning.